test/WebSites/BasicWebSite/wwwroot/javascript/jquery.validate.unobtrusive.3.2.6.js

```diff
@@ -0,0 +1,416 @@
+/*!
```

Comment:
This file doesn't normally have the version in the name, does it? Can we instead acquire it through Bower/Grunt? @pranavkm has done some stuff related to this in the past.